src/types/HTTPError.ts

```diff
@@ -41,7 +41,7 @@ export enum HTTPError {
   SITE_AREA_WITH_NO_SITE_ERROR = 525,
   USER_WITH_NO_SITE_ERROR = 525,
 
-  BILLING_DELETE_ERROR = 510, // Override an already defined status code: Not extended (RFC 2774)
+  BILLING_DELETE_ERROR = 510, // FIXME: override an already defined HTTP status code: Not extended (RFC 2774)
```

Comment:
No need to be unique, each code will be used in a dedicated area (Billing...) as we have only 99 errors possible (500-599.)
Only general HTTP Code should be.

Reply:
The chosen code shall not override an already existing HTTP status code with a complete different meaning. It shall reuse an already existing with the same meaning or define its own not yet reserved then.

Reply:
> The chosen code shall not override an already existing HTTP status code with a complete different meaning. It shall reuse an already existing with the same meaning or define its own not yet reserved then.

And where there will be all code taken what will you do!?

Reply:
HTTP code shall only be used to return HTTP protocol related behaviour ;) 
If you need to return application specific errors, an error message shall be returned with application own error code and error message, like in OCPP for example.

Reply:
> HTTP code shall only be used to return HTTP protocol related behaviour ;)
> If you need to return application specific errors, an error message shall be returned with application own error code and error message in the body, like in OCPP for example.

Okay, then we have another place for a small refactoring then :-).